modules/test/playwright/tests/users-admin-web/main/users.spec.ts

```diff
@@ -246,3 +246,119 @@ test(
 		).toBeVisible();
 	}
 );
+
+test(
+	'Cannot create user with invalid screen name',
+	{tag: '@LPD-57460'},
+	async ({editUserPage, usersAndOrganizationsPage}) => {
+		const testScreenName = async (
+			screenName: string,
+			screenNameVariation: string
+		) => {
+			await editUserPage.screenNameInput.fill(screenName);
+			await editUserPage.saveButton.click();
+
+			if (screenNameVariation === 'empty') {
+				await expect(editUserPage.screenNameError).toHaveText(
+					'The Screen Name field is required.'
+				);
+			}
+			else if (screenNameVariation === 'valid') {
+				await expect(editUserPage.screenNameError).not.toBeVisible();
+			}
+			else {
+				await expect(editUserPage.screenNameError).toHaveText(
```

Comment:
can we just use just toContainsText with a smaller significant part? like `The screen name cannot be an email address or a reserved word`